modules/apps/27-interchain-accounts/controller/keeper/migrations.go

```diff
@@ -0,0 +1,39 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	capabilitytypes "github.com/cosmos/cosmos-sdk/x/capability/types"
+
+	"github.com/cosmos/ibc-go/v5/modules/core/23-commitment/types"
+	host "github.com/cosmos/ibc-go/v5/modules/core/24-host"
+)
+
+// Migrator is a struct for handling in-place store migrations.
+type Migrator struct {
+	keeper *Keeper
+}
+
+// NewMigrator returns a new Migrator.
+func NewMigrator(keeper *Keeper) Migrator {
+	return Migrator{keeper: keeper}
+}
+
+// AssertChannelCapabilityMigrations checks that all channel capabilities generated using the interchain accounts controller port prefix
+// are owned by the controller submodule and ibc.
+func (m Migrator) AssertChannelCapabilityMigrations(ctx sdk.Context) error {
```

Comment:
I'm open to suggestions on improving the name of this method.

Reply:
I like the name